core/src/main/java/org/apache/accumulo/core/metadata/schema/CompactionMetadata.java

```diff
@@ -121,7 +122,7 @@ public static CompactionMetadata fromJson(String json) {
     return new CompactionMetadata(jData.inputs.stream().map(StoredTabletFile::new).collect(toSet()),
         StoredTabletFile.of(jData.tmp).getTabletFile(), jData.compactor,
         CompactionKind.valueOf(jData.kind), jData.priority,
-        CompactorGroupIdImpl.groupId(jData.groupId), jData.propDels, jData.fateTxId);
+        CompactorGroupIdImpl.groupId(jData.groupId), jData.propDels, jData.fateId);
```

Comment:
This suggestion goes with another comment.

```suggestion
        CompactorGroupIdImpl.groupId(jData.groupId), jData.propDels, FateId.from(jData.fateId));
```